### PR TITLE
New frontend

### PR DESCRIPTION
**Pull request by Lev-Stambler**

### CODE REVIEW

**Comment by shivaPeri** [comment]
Fixed merge conflicts
@Lev-Stambler review `/frontend`

File: frontend/src/App.svelte [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<!-- <script land="ts">
```
Comment (Lev-Stambler) [inline frontend/src/App.svelte:1]: Pls delete comments

File: frontend/public/index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	<meta charset='utf-8'>
 	<meta name='viewport' content='width=device-width,initial-scale=1'>
 
-	<title>3D Turtles</title>
+	<title>Svelte app</title>
```
Comment (Lev-Stambler) [inline frontend/public/index.html:7]: Fine for now but we gotta come up with a final name...

File: frontend/src/components/Controls.svelte [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	<button on:click={togglePreview}
+		>{$controlParams.displayPreview ? 'show preview' : 'hide preview'}</button
+	>
+	<input type="range" bind:value={$controlParams.pathWidth} min="0.01" max="10" step=".01" />
+
+	<!-- TODO: control this from URL param -->
+	{#if $controlParams.allowControls}
+		<RationalButtonGroup name="Yaw" bind:rational={yaw} on:paramchange={updateParams} />
+		<RationalButtonGroup name="Pitch" bind:rational={pitch} on:paramchange={updateParams} />
+		<!-- <RationalButtonGroup name="Distance" bind:rational={distance} on:paramchange={updateParams} /> -->
```
Comment (Lev-Stambler) [inline frontend/src/components/Controls.svelte:48]: commented code

File: frontend/src/components/Controls.svelte [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+<section>
+	<button on:click={updateParams}>Reset</button>
+	<button on:click={toggleRunning}>{$controlParams.running ? 'Pause' : 'Play'}</button>
+	<button on:click={togglePreview}
+		>{$controlParams.displayPreview ? 'show preview' : 'hide preview'}</button
+	>
+	<input type="range" bind:value={$controlParams.pathWidth} min="0.01" max="10" step=".01" />
+
+	<!-- TODO: control this from URL param -->
```
Comment (Lev-Stambler) [inline frontend/src/components/Controls.svelte:44]: Can you link to a GitHub Issue for this?

**Comment by Lev-Stambler** [comment]
> Fixed merge conflicts @Lev-Stambler review `/frontend`

Looks good pending a few small comments @shivaPeri

**Pull request merged**